non-nix-build/build_in_simple_env.sh

```diff
@@ -38,7 +38,10 @@ bash build_db_backend.sh
 
 # setup/copy/link other files
 cp $ROOT_DIR/resources/electron $DIST_DIR/bin/
-cp $(which ruby) $DIST_DIR/bin/ruby
+
+# The built-in macOS ruby binary is too old and has to be hacked around
+cp $(brew --prefix ruby)/bin/ruby $DIST_DIR/bin/ruby
```

Comment:
sounds good: i only find https://github.com/rbenv/rbenv/issues/343#issuecomment-14008198 as a possible problem: is it an issue, or would we maybe hope to think of a more stable solution longterm